server/src/main/scala/org/http4s/server/ServerBuilder.scala

```diff
@@ -29,6 +29,10 @@ trait ServerBuilder {
   def withExecutorService(executorService: ExecutorService): Self =
     withExecutionContext(ExecutionContext.fromExecutorService(executorService))
 
+  @deprecated("Use withExecutionContext", "0.17.0")
```

Comment:
This existed in 0.16 and was taken away and tripped up a user.  It tripped me up merging from 0.16.  This is just compiler-guided migration.